bleachbit/Winapp.py

```diff
@@ -134,9 +134,10 @@ def special_detect(code):
 
 
 def fnmatch_translate(pattern):
-    """Same as the original without the end"""
+    """Same as the original without the end and escaping square brackets"""
     import fnmatch
-    ret = fnmatch.translate(pattern)
+    ret = re.sub('[][]', lambda m: '[%s]' % m.group(0), pattern)
```

Comment:
@livingentity 
You can take a look at the format method of the string class in Python and use it in the lambda function instead of the old style with %.

Reply:
OK, but does that make any difference?